src/main/java/com/devathon/preguntonic/model/Room.java

```diff
@@ -23,7 +25,9 @@ public class Room {
   private RoomStatus status;
   private Game game;
   private int maxPlayers;
-  private int currentPlayers;
   private int numQuestions;
   private LocalDateTime createdAt;
+
+  private final Map<UUID, Player> players = Map.of();
```

Comment:
This can cause conflicts, but I can use Map instead of List

Reply:
I was referring to the change in this [PR](https://github.com/Devathon-2024-team5/Preguntonic-backend/pull/9/files) in that I start using only a list, but I can change it in the other PR

Reply:
Thats true... i implement it on this way to execute the life cycle of the game. We can use anyone, but i prefer Map because is O(1) on "get" operations